test/broadcast.jl

```diff
@@ -244,6 +247,28 @@ let x = [1:4;]
     @test sin.(f17300kw.(x, y=1)) == sin.(f17300kw.(x; y=1)) == sin.(x .+ 1)
 end
 
+# splice escaping of @.
+let x = [4, -9, 1, -16]
+    @test [2, 3, 4, 5] == @.(1 + sqrt($sort(abs(x))))
```

Comment:
If I'm not mistaken, I think in this case the macro is just being called with the parenthesized form, in which case there shouldn't be a space after `@.`.